dist/index.html

```diff
@@ -4,8 +4,11 @@
 		<meta charset="utf-8">
 		<title>Development</title>
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
+		<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css‌​/font-awesome.min.cs‌​s"> -->
 		<link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
 		<link href="https://fonts.googleapis.com/css?family=Scope+One" rel="stylesheet">
```

Comment:
@sarjford We should try to use less than 2 types of Google fonts as slows down page rendering.